plugins/admission/cmd/admission/main.go

```diff
@@ -33,21 +34,14 @@ func exitWithError(message string, err error) {
 	}
 }
 
-var handler fadmission.Validator
```

Comment:
I removed package vars usage and env vars usage scattered over the app, instead the functions now depend on the `InsightsConfig` struct, I think this will make test easier